<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Tools and statistical approaches for integrating DNA sequencing into clinical care</title>
  <meta name="description" content="Dissertation by Dayne Filer for the degree of Doctorate of Philosophy in the UNC School of Medicine Curriculum in Bioinformatics and Computational Biology." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Tools and statistical approaches for integrating DNA sequencing into clinical care" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://daynefiler.com/dissertation/_book/" />
  
  <meta property="og:description" content="Dissertation by Dayne Filer for the degree of Doctorate of Philosophy in the UNC School of Medicine Curriculum in Bioinformatics and Computational Biology." />
  <meta name="github-repo" content="daynefiler/dissertation" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Tools and statistical approaches for integrating DNA sequencing into clinical care" />
  
  <meta name="twitter:description" content="Dissertation by Dayne Filer for the degree of Doctorate of Philosophy in the UNC School of Medicine Curriculum in Bioinformatics and Computational Biology." />
  

<meta name="author" content="Dayne Lewis Filer" />


<meta name="date" content="2020-01-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="intro.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Front matter</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#copyright"><i class="fa fa-check"></i>Copyright</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#dedication"><i class="fa fa-check"></i>Dedication</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#preface"><i class="fa fa-check"></i>Preface</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#outline"><i class="fa fa-check"></i><b>1.1</b> Outline</a></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#humGenPrime"><i class="fa fa-check"></i><b>1.2</b> Human Genetics Primer</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#dnaDiscover"><i class="fa fa-check"></i><b>1.2.1</b> Discovery of DNA and the central dogma</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#genVar"><i class="fa fa-check"></i><b>1.2.2</b> Genetic variation in humans</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#sequencing"><i class="fa fa-check"></i><b>1.3</b> DNA Sequencing</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#firstGen"><i class="fa fa-check"></i><b>1.3.1</b> First-generation sequencing</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#secGen"><i class="fa fa-check"></i><b>1.3.2</b> Second-generation sequencing</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#illumina"><i class="fa fa-check"></i><b>1.3.3</b> Illumina sequencing</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#seqData"><i class="fa fa-check"></i><b>1.3.4</b> Processing short-read sequencing data</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#medGen"><i class="fa fa-check"></i><b>1.4</b> Development of Medical Genetics</a></li>
<li class="chapter" data-level="1.5" data-path="intro.html"><a href="intro.html#prob"><i class="fa fa-check"></i><b>1.5</b> Rationale</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="mcCNV.html"><a href="mcCNV.html"><i class="fa fa-check"></i><b>2</b> A novel copy number variant algorithm</a><ul>
<li class="chapter" data-level="2.1" data-path="mcCNV.html"><a href="mcCNV.html#introduction"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="mcCNV.html"><a href="mcCNV.html#methods"><i class="fa fa-check"></i><b>2.2</b> Methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="mcCNV.html"><a href="mcCNV.html#exome-sequencing"><i class="fa fa-check"></i><b>2.2.1</b> Exome sequencing</a></li>
<li class="chapter" data-level="2.2.2" data-path="mcCNV.html"><a href="mcCNV.html#genome-sequencing"><i class="fa fa-check"></i><b>2.2.2</b> Genome sequencing</a></li>
<li class="chapter" data-level="2.2.3" data-path="mcCNV.html"><a href="mcCNV.html#simulating-targeted-sequencing"><i class="fa fa-check"></i><b>2.2.3</b> Simulating targeted sequencing</a></li>
<li class="chapter" data-level="2.2.4" data-path="mcCNV.html"><a href="mcCNV.html#mccnv-algorithm"><i class="fa fa-check"></i><b>2.2.4</b> mcCNV algorithm</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="mcCNV.html"><a href="mcCNV.html#results"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="mcCNV.html"><a href="mcCNV.html#multiplexed-capture-reduces-inter-sample-variance"><i class="fa fa-check"></i><b>2.3.1</b> Multiplexed capture reduces inter-sample variance</a></li>
<li class="chapter" data-level="2.3.2" data-path="mcCNV.html"><a href="mcCNV.html#multiplexed-capture-provides-controls-for-exomedepth"><i class="fa fa-check"></i><b>2.3.2</b> Multiplexed capture provides controls for ExomeDepth</a></li>
<li class="chapter" data-level="2.3.3" data-path="mcCNV.html"><a href="mcCNV.html#mccnv-exomedepth-perform-comparably-in-simulation-study"><i class="fa fa-check"></i><b>2.3.3</b> mcCNV &amp; ExomeDepth perform comparably in simulation study</a></li>
<li class="chapter" data-level="2.3.4" data-path="mcCNV.html"><a href="mcCNV.html#mccnv-exomedepth-perform-comparably-on-wgs-pool"><i class="fa fa-check"></i><b>2.3.4</b> mcCNV &amp; ExomeDepth perform comparably on WGS pool</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="mcCNV.html"><a href="mcCNV.html#discussion"><i class="fa fa-check"></i><b>2.4</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="cfES.html"><a href="cfES.html"><i class="fa fa-check"></i><b>3</b> Shortcomings of exome sequencing in noninvasive prenatal genetics</a><ul>
<li class="chapter" data-level="3.1" data-path="cfES.html"><a href="cfES.html#introduction-1"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="cfES.html"><a href="cfES.html#methods-1"><i class="fa fa-check"></i><b>3.2</b> Methods</a><ul>
<li class="chapter" data-level="3.2.1" data-path="cfES.html"><a href="cfES.html#participant-selection"><i class="fa fa-check"></i><b>3.2.1</b> Participant selection</a></li>
<li class="chapter" data-level="3.2.2" data-path="cfES.html"><a href="cfES.html#exome-sequencing-and-analysis"><i class="fa fa-check"></i><b>3.2.2</b> Exome sequencing and analysis</a></li>
<li class="chapter" data-level="3.2.3" data-path="cfES.html"><a href="cfES.html#genotyping-algorithm"><i class="fa fa-check"></i><b>3.2.3</b> Genotyping algorithm</a></li>
<li class="chapter" data-level="3.2.4" data-path="cfES.html"><a href="cfES.html#data-availability"><i class="fa fa-check"></i><b>3.2.4</b> Data availability</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="cfES.html"><a href="cfES.html#results-1"><i class="fa fa-check"></i><b>3.3</b> Results</a></li>
<li class="chapter" data-level="3.4" data-path="cfES.html"><a href="cfES.html#discussion-1"><i class="fa fa-check"></i><b>3.4</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="future-directions.html"><a href="future-directions.html"><i class="fa fa-check"></i><b>4</b> Future Directions</a><ul>
<li class="chapter" data-level="4.1" data-path="future-directions.html"><a href="future-directions.html#fetal-genotyping-project"><i class="fa fa-check"></i><b>4.1</b> Fetal genotyping project</a></li>
<li class="chapter" data-level="4.2" data-path="future-directions.html"><a href="future-directions.html#cnv-validation-proposal"><i class="fa fa-check"></i><b>4.2</b> CNV validation proposal</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="r-code.html"><a href="r-code.html"><i class="fa fa-check"></i><b>A</b> R Code</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Tools and statistical approaches for integrating DNA sequencing into clinical care</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Tools and statistical approaches for integrating DNA sequencing into clinical care</h1>
<p class="author"><em>Dayne Lewis Filer</em></p>
<p class="date"><em>2020</em></p>
</div>
<div id="front-matter" class="section level1 unnumbered">
<h1>Front matter</h1>
<div id="copyright" class="section level2 unnumbered">
<h2>Copyright</h2>
<p>© 2020 Dayne Lewis Filer, ALL RIGHTS RESERVED</p>
</div>
<div id="abstract" class="section level2 unnumbered">
<h2>Abstract</h2>
<p>The discovery of DNA fundamentally changed the world, revolutionizing our understanding of life and the practice of medicine.
After a century of studying DNA, medicine entered a new frontier with the completion of the nearly 20-year billion-dollar effort to sequence the first human genome.
We can now sequence a human genome in a matter of days for hundreds (not billions) of dollars.
Technological advances and medical geneticists’ robust efforts to interpret human variation have led to exponential clinical sequencing growth.
The medical genetics community currently faces three primary challenges: (1) variant interpretation; (2) overcoming difficult detection problems (e.g., structural variants and low-frequency variants); (3) moving beyond a linear poorly-representative reference genome.
The work herein addresses how to overcome two specific detection problems.</p>
<p>First, I present a novel approach for detecting exon-level copy number variation using exome sequencing.
The vast majority of available sequencing collected lacks the power to detect small copy number variants, leading to a significant blind spot in our understanding of genetic variation.
I demonstrate how modifying the exome capture step to capture multiple samples simultaneously significantly reduces the inter-sample variance and improves copy number discrimination.
I then demonstrate the utility of a novel statistical algorithm specifically for multiplexed-capture exome sequencing.</p>
<p>Second, I outline the shortcomings of noninvasive exome sequencing in prenatal genetics.
Utilizing cell-free fetal DNA in maternal circulation, we can diagnose a wide range of genetic conditions noninvasively.
Efforts have suggested the possibility of noninvasive fetal genome and exome sequencing.
However, to-date, no one has demonstrated accurate fetal genotyping purely from cell-free DNA.
I use probability theory to demonstrate why efforts have failed, and suggest a path forward for noninvasive fetal genotyping.</p>
<p>Finally, I briefly outline my ongoing work in prenatal genetics and propose a validation study to further interrogate exon-level copy number variation.</p>
</div>
<div id="dedication" class="section level2 unnumbered">
<h2>Dedication</h2>
<p><em>To my loving and endlessly supportive wife and parents,</em></p>
<p><em>it’s all happening…</em></p>
<p><br>

</p>
<p><em>The only true currency in this bankrupt world</em></p>
<p><em>is what you share with someone else</em></p>
<p><em>when you’re uncool.</em></p>
<p>– Cameron Crowe</p>
</div>
<div id="acknowledgments" class="section level2 unnumbered">
<h2>Acknowledgments</h2>
<p>This work would not have been possible without the support of my PhD mentor, Kirk Wilhelmsen.
Kirk provided mentorship, advice, and care in and out of the laboratory.
Most importantly, Kirk encouraged me to pursue research and avenues to best serve my career – even when my path did not further his own research program.</p>
<p>My scientific training began with Sara Heggland at the College of Idaho.
I spent four years working under Sara’s mentorship; she, more than anyone, taught me the scientific method and galvanized my path to research.
I am forever grateful she took me under her wing.
After graduating from college, David Reif, Richard Judson, Matthew Martin, and others at the National Center for Computational Toxicology allowed me to find my passion for data science.
Working at NCCT gave me the practical skills to thrive in a data science PhD.</p>
<p>Thank you to Stan Ahalt and all of the people at the Renaissance Computing Institute for welcoming and supporting me, particularly Chris Bizon and Phil Owen.
Thank you to my committee, Bradford Powell, Stan Ahalt, Neeta Vora, and Yun Li.
In addition, I want to thank all of my collaborators for this work, especially: Jonathan Berg, Alicia Brandt, Piotr Mieczkowski, Karen Dorman, Amber Ivins, and Fengshen Kuo.</p>
<p>Thank you to the UNC MD-PhD program for accepting and supporting me both through medical school and my PhD training.</p>
<p>Finally, thank you to my family and friends whose support outside of training have enabled and enriched my life and success.</p>
</div>
<div id="preface" class="section level2 unnumbered">
<h2>Preface</h2>
<p>I have submitted portions of Chapters <a href="mcCNV.html#mcCNV">2</a> and <a href="cfES.html#cfES">3</a> for peer-reviewed publication; both manuscripts are under review currently.
Both manuscripts represent a collaborative effort, particularly in the data collection, but I independently performed all analyses, produced the text, code, figures, and tables herein.
The manuscript derived from Chapter <a href="mcCNV.html#mcCNV">2</a> includes the following authorship: Dayne L Filer, Fengshen Kuo, Alicia T Brandt, Christian R Tilley, Piotr A Mieczkowski, Kimberly Robasky, Chris Bizon, Jeffery L Tilson, Bradford C Powell, Darius M Bost, Clark D Jeffries, Jonathan S Berg, and Kirk C Wilhelmsen.
The manuscript dervied from Chapter <a href="cfES.html#cfES">3</a> includes the following authorship: Dayne L Filer, Piotr A Mieczkowski, Alicia Brandt, Kelly L Gilmore, Bradford C Powell, Jonathan S Berg, Kirk C Wilhelmsen, and Neeta L Vora.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="intro.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": true,
"twitter": true,
"linkedin": true,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["DLFilerDissertation.pdf"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
